examples/pipelines/datacomp/components/clean_captions/fondant_component.yaml

```diff
@@ -1,6 +1,6 @@
 name: Clean captions
 description: Component that filters out bad captions (Empty captions, Captions with weird characters, Captions that are dates)
-image: ghcr.io/ml6team/clean_captions:50f3a97878ac81670ebe624039ff0fcec0542e4f
+image: fndnt/clean_captions:50f3a97878ac81670ebe624039ff0fcec0542e4f
```

Comment:
This one I can't see on docker hub. I think the example pipeline images are not pushed to docker hub?